src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -173,6 +174,7 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
 
                 override fun scanningIacFinished(iacResult: IacResult) {
                     currentIacResult = iacResult
+                    iacCacheDirty = false
```

Comment:
scanningIacError below need that `iacCacheDirty = false` too

Reply:
Oh yeah, else we could launch several scans.